charts/renku/values.yaml

```diff
@@ -528,6 +528,12 @@ ui:
   # templatesRepository:
     # url: https://github.com/SwissDataScienceCenter/renku-project-template
     # ref: 0.1.5
+  ## Configuration for Sentry
+  # sentry:
+  #   enabled: true
```

Comment:
is it on by default? The commented values here should show the defaults.

Reply:
oh I see, let me change it then